build/yamls/antrea.yml

```diff
@@ -6947,7 +6947,7 @@ spec:
       serviceAccountName: antrea-agent
       initContainers:
         - name: install-cni
-          image: "antrea/antrea-ubuntu:latest"
+          image: "antrea/antrea-agent-ubuntu:latest"
```

Comment:
this PR may benefit from being broken into 2 parts. The reason is that after we merge this PR, `antrea/antrea-agent-ubuntu:latest` and `antrea/antrea-controller-ubuntu:latest` will not exist in the registry for a while. The workflows in this repository only push the amd64 version of the images: `antrea/antrea-agent-ubuntu-amd64:latest` and `antrea/antrea-controller-ubuntu-amd64:latest`. We have a separate workflow in a separate repository which takes care of the arm images AND takes care of pushing the multi-arch manifests (`antrea/antrea-agent-ubuntu:latest` and `antrea/antrea-controller-ubuntu:latest`).

So if we merge this PR as is, and someone tries to apply https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/antrea.yml, I expect that it will fail.

The steps should probably be as follows:
1) all the infrastructure changes required to build the split images are merged, but the Helm chart and YAML manifests are not modified yet
2) I take care of updating the infrastructure which builds and pushes the ARM images and multi-arch manifests
3) changes to the Helm chart and YAML manifests are merged

cc @luolanzone for visibility

Reply:
sure, so for this PR I can remove the YAML changes and the helm chart changes?

Reply:
Yes. Remove them from this PR but keep them for the follow-up PR.
As soon as the 1st PR is merged, I'll update the build infra.

Reply:
sure

Reply:
also the generate-manifest changes have to be reverted?

Reply:
but won't that affect the github and other ci jobs because with the unmodified yaml it will test the unified image and not the split images.
@antoninbas

Reply:
Created two different PRs for the same, [this](https://github.com/antrea-io/antrea/pull/5902) PR is to build the images, and [this](https://github.com/antrea-io/antrea/pull/5903) one is to update the yamls, ci and github workflows.